### PR TITLE
Add business support CTA to admin panel

### PR DESCRIPTION
**Pull request by bitbager**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| License         | MIT                                                          |

### CODE REVIEW

**Comment by diimpp** [comment]
image
At least it needs indication icon of `external url`
image

What about this section? Looks like new URLs should be placed over here firstly. 
image

File: src/Sylius/Bundle/AdminBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -47,6 +47,11 @@ sylius:
                 sales:
                     header: Ventes
                     orders: Commandes
+                support:
+                    header: Soutien
```
Comment (Jibbarth) [inline src/Sylius/Bundle/AdminBundle/Resources/translations/messages.fr.yml]: `Support` can/should be used in this context

Comment (loic425) [reply src/Sylius/Bundle/AdminBundle/Resources/translations/messages.fr.yml]: I agree, Support should be used in this case.

**Comment by bitbager** [comment]
> image At least it needs indication icon of `external url` image
> 
> What about this section? Looks like new URLs should be placed over here firstly. image

Valid points. I changed the menu item group completely and added the target="_blank" to each link

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "master"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by GSadee** [comment]
Closing in favour of #15642

**Pull request closed without merging**